managed/models/database.go

```diff
@@ -893,6 +894,10 @@ func SetupDB(sqlDB *sql.DB, params *SetupDBParams) (*reform.DB, error) {
 
 	if pErr, ok := errDB.(*pq.Error); ok && pErr.Code == "28000" {
 		// invalid_authorization_specification	(see https://www.postgresql.org/docs/current/errcodes-appendix.html)
+		if host, _, _ := net.SplitHostPort(params.Address); host != "127.0.0.1" {
```

Comment:
Is there a better way to do this?
This is non-deterministic. The fact the host is `127.0.0.1` does not mean we do or don't have access to `root`.

Reply:
generalized error added

Reply:
after discussion in slack, it was decided to add user friendly message on auth fail and attempt to create user and db.